src/bgw/job_stat.c

```diff
@@ -19,6 +19,9 @@
 #define MAX_FAILURES_MULTIPLIER 20
 #define MIN_WAIT_AFTER_CRASH_MS (5 * 60 * 1000)
 
+#define JOB_STAT_FLAGS_DEFAULT 0
+#define LAST_CRASH_REPORTED 1
+
```

Comment:
Might be better to add in the header file, so other dependent modules can use it.